common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -36,6 +36,11 @@ object Messages {
     deviceId: Option[DeviceId],
     deviceType: DeviceType) extends BusMessage
 
+  final case class DevicePublicCredentialsSet(
```

Comment:
This was credentials before, but credentials makes it look like they are secret and apparently these are just public keys